File: PHP/Cap05/c05ex11.php
<?php
// c05ex11.php

for ($I = 1; $I <= 5; $I++) {
  printf("Entre um valor numerico: ");
  $N = intval(trim(fgets(STDOUT)));

  $R = $N * 3;

  printf("Resultado = %d\n\n", $R);
}

printf("\n");
printf("Tecle <Enter> para encerrar... ");
trim(fgets(STDOUT));

?>

File: RUBY/Cap05/c05ex11.rb
# c05ex11.rb

#!/usr/bin/env ruby

for i in 1.step(by: 1, to: 5)
  print "Entre um valor numerico: "
  n = gets.chomp.to_i

  r = n * 3

  puts "Resultado = #{r}"
  puts
end

puts
print "Tecle <Enter> para encerrar... "
gets
